docs/implementation.rst

```diff
@@ -0,0 +1,5 @@
+Build an org implementation
```

Comment:
This file incomplete?

Reply:
Yes. This can be removed. I have it down to work on after the initial release.